project_generator/templates/makefile_gcc.tmpl

```diff
@@ -175,6 +175,7 @@ clean:
 	$(RM) $(TARGET).$(TARGET_EXT) $(TARGET).bin $(TARGET).map $(OBJ_FOLDER)*.* $(OBJ_FOLDER)
 	@echo ' '
 
-
+{% block extra_targets %}
```

Comment:
Nothing yet. I was going to use it. I'll remove it.
On May 21, 2016 12:27 PM, "Martin Kojtal" <notifications@github.com> wrote:

> In project_generator/templates/makefile_gcc.tmpl
> <https://github.com/project-generator/project_generator/pull/384#discussion_r64137606>
> :
>
> > @@ -175,6 +175,7 @@ clean:
> >  	$(RM) $(TARGET).$(TARGET_EXT) $(TARGET).bin $(TARGET).map $(OBJ_FOLDER)*.* $(OBJ_FOLDER)
> >  	@echo ' '
> >
> > -
> > +{% block extra_targets %}
>
> what does extra_targets do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/project-generator/project_generator/pull/384/files/763ae4943f00a50becd69922896bbba270594eb4#r64137606>
>